src/register_version.cpp

```diff
@@ -1,12 +0,0 @@
-// Copyright 2018-2019 Henry Schreiner and Hans Dembinski
-//
-// Distributed under the 3-Clause BSD License.  See accompanying
-// file LICENSE or https://github.com/scikit-hep/boost-histogram for details.
-
-#include <boost/histogram/python/pybind11.hpp>
-
-#include <boost/histogram/histogram.hpp>
-
-void register_version(py::module &m) {
-    m.attr("BOOST_HISTOGRAM_DETAIL_AXES_LIMIT") = BOOST_HISTOGRAM_DETAIL_AXES_LIMIT;
```

Comment:
What about restoring this as `"_AXES_LIMIT"`? Then using it in the histogram creation function to check the maximum number of axes to fix #128?

Reply:
That would be ok. I am also ok with calling it `_axes_limit`.

Reply:
Why not making the check in C++, where you have access to `*AXES_LIMIT`?

Reply:
Because in #130 I move histogram and make_histogram to Python. The only thing left would be to add it to each histogram constructor, replacing `py::init<...>()` with custom code that returns a pointer. Not awful, but not necessary quite as nice.

Reply:
Let me know which way you want to go. I will add `_axes_limit` in this PR if you want the check to happen in Python. If you fix this is Boost.Histogram, we can just rely on that exception for now.

Reply:
I'll go this way for now, and change later if you want me to.